lib/design_system/graphics/icons/icon_size.dart

```diff
@@ -0,0 +1,10 @@
+enum IconSize {
+  pt16(16.0),
```

Comment:
The term "Foundation" here refers to the Design System's Foundation (typography, colors, spacing, etc) rather than a "code" foundation, and since this enum defines the allowed icon-sizes, I thought that it made most sense to place it locally together with the other icon-related types.  

<img width="202" alt="Foundation" src="https://github.com/Concordium/concordium-wallet/assets/152864167/982fa21e-c342-46e9-9b28-83c0c0819d79">